docs/modules/ROOT/pages/release-notes.adoc

```diff
@@ -1,186 +1,38 @@
-= 5.2 Release Notes
+= 5.3 Release Notes
 
-In this release, we introduced the Backup and Restore feature with support for PVC and hostPath. This feature uses xref:5.1@hazelcast:storage:persistence.adoc[Hazelcast Persistence] under the hood.
+In this release, we have introduced Map Custom Resource and external storage support for Backup & Restore.
```

Comment:
```suggestion
In this release, we introduced a xref:map-configuration.adoc[Custom Resource for Hazelcast maps] as well as support for xref:backup-restore.adoc[backing up map data to cloud storage].
```

Reply:
Thanks for the suggestion, Jake!  But our goal is to automate the documentation process in the nearest future and in case of adding the links into a summary it will be difficult to automate this process. That's why we left the summary without any reference to documents. So what do you think if we leave it as it is? Or we can release 5.3 with xref links but the upcoming release will be without these xref tags.

Reply:
I think let's include the links for this release. We can think about how to deal with future releases together.

Automated release notes would be a great improvement to our process. Are we taking pull request titles/commits and using those as release notes? if so, we'll need to make sure we have a style guide for commit messages so that the release notes are consistent.